src/modules/dashboard/components/ColonyMembers/ColonyMembers.tsx

```diff
@@ -1,159 +1,49 @@
-    if (
-      !colonySubscribedUsers ||
-      !colonySubscribedUsers.colony.subscribedUsers.length
-    ) {
-      return 0;
-    }
-    const {
-      colony: { subscribedUsers },
-    } = colonySubscribedUsers;
-    if (subscribedUsers.length <= MAX_AVATARS) {
-      return 0;
-    }
-    return subscribedUsers.length - MAX_AVATARS;
-  }, [colonySubscribedUsers]);
-
-  if (!colonySubscribedUsers || loadingColonySubscribedUsers) {
-    return (
-      <div className={styles.main}>
-        <Heading
-          appearance={{ size: 'normal', weight: 'bold' }}
-          text={MSG.title}
-          textValues={{ hasCounter: false }}
-        />
-        <SpinnerLoader appearance={{ size: 'small' }} />
-        <span className={styles.loadingText}>
-          <FormattedMessage {...MSG.loadingData} />
-        </span>
-      </div>
-    );
-  }
-
-  const {
-    colony: { subscribedUsers },
-  } = colonySubscribedUsers;
-
   return (
     <div className={styles.main}>
-      <NavLink
-        /*
-         * @TODO Put in the Community route, once that is created in DEV-13
-         */
-        to={DASHBOARD_ROUTE}
-      >
-        <Heading
-          appearance={{ size: 'normal', weight: 'bold' }}
-          text={MSG.title}
-          textValues={{ count: subscribedUsers.length, hasCounter: true }}
-        />
-      </NavLink>
-      <ul className={styles.userAvatars}>
-        {(subscribedUsers as AnyUser[])
-          .slice(0, avatarsDisplaySplitRules)
-          .map((user) => (
-            <li className={styles.userAvatar} key={user.id}>
-              <UserAvatar
-                size="xs"
-                colonyAddress={colonyAddress}
-                address={user.profile.walletAddress}
-                user={user}
-                showInfo
-                notSet={false}
-                popperProps={{
-                  placement: 'bottom',
-                  showArrow: false,
-                  modifiers: [
-                    {
-                      name: 'offset',
-                      options: {
-                        /*
-                         * @NOTE Values are set manual, exactly as the ones provided in the figma spec.
-                         *
-                         * There's no logic to how they are calculated, so next time you need
-                         * to change them you'll either have to go by exact specs, or change
-                         * them until it "feels right" :)
-                         */
-                        offset: [-208, -12],
-                      },
-                    },
-                  ],
-                }}
-              />
-            </li>
-          ))}
-        {!!remainingAvatarsCount && (
-          <li className={styles.remaningAvatars}>
-            {remainingAvatarsCount < 99 ? remainingAvatarsCount : `>99`}
-          </li>
-        )}
-      </ul>
+      <div className={styles.colonyList}>
+        <SubscribedColoniesList />
```

Comment:
With the changed navigation "theme" wrapper, this doesn't really make sense on the page anymore, plus it looks kinda weird.

Remove it please.

If product complains, we'll deal with it then.